### ISSUE TITLE
Tutorial doesn't work

### ISSUE THREAD

**Issue by ajandria** [Jiaxin-Fan/MuSiC2#4]
Hello @Jiaxin-Fan,

I need help with the error your tutorial is giving me

```
> # music2 deconvolution
> set.seed(1234)
> est.prop = music2_prop(
+   bulk.eset = benchmark.eset,
+   sc.eset = seger.eset,
+   condition = 'group',
+   control = 'healthy',
+   case = 't2d',
+   clusters = 'cellType',
+   samples = 'sampleID',
+   select.ct = c('acinar', 'alpha', 'beta', 'delta', 'ductal', 'gamma'),
+   n_resample = 20,
+   sample_prop = 0.5,
+   cutoff_c = 0.05,
+   cutoff_r = 0.01
+ )$Est.prop
Error in rownames(bulk.mtx) : 
  argument "bulk.mtx" is missing, with no default
```

**Comment by xoelmb** [comment]
I have just installed MuSiC2 and I am encountering this same error.

> R version 4.2.1 (2022-06-23)
> Platform: x86_64-conda-linux-gnu (64-bit)
> Running under: Ubuntu 20.04.5 LTS
> 
> Matrix products: default
> BLAS/LAPACK: /users/genomics/xoel/micromamba/envs/music2/lib/libopenblasp-r0.3.21.so
> 
> locale:
>  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       
>  [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   
>  [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          
> [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   
> 
> attached base packages:
> [1] stats     graphics  grDevices utils     datasets  methods   base     
> 
> other attached packages:
> [1] MuSiC2_0.1.0        Biobase_2.56.0      BiocGenerics_0.42.0
> 
> loaded via a namespace (and not attached):
>   [1] mcmc_0.9-7                  bitops_1.0-7               
>   [3] matrixStats_0.62.0          MuSiC_1.0.0                
>   [5] bit64_4.0.5                 httr_1.4.4                 
>   [7] doParallel_1.0.17           RColorBrewer_1.1-3         
>   [9] GenomeInfoDb_1.32.4         repr_1.1.4                 
>  [11] backports_1.4.1             tools_4.2.1                
>  [13] utf8_1.2.2                  R6_2.5.1                   
>  [15] DBI_1.1.3                   colorspace_2.0-3           
>  [17] withr_2.5.0                 tidyselect_1.2.0           
>  [19] GGally_2.1.2                xbioc_0.1.19               
>  [21] bit_4.0.4                   compiler_4.2.1             
>  [23] cli_3.4.1                   quantreg_5.94              
>  [25] SparseM_1.81                pkgmaker_0.32.2.900        
>  [27] DelayedArray_0.22.0         checkmate_2.1.0            
>  [29] scales_1.2.1                nnls_1.4                   
>  [31] quadprog_1.5-8              proxy_0.4-27               
>  [33] pbdZMQ_0.3-7                stringr_1.4.1              
>  [35] digest_0.6.30               MCMCpack_1.6-3             
>  [37] XVector_0.36.0              base64enc_0.1-3            
>  [39] pkgconfig_2.0.3             htmltools_0.5.3            
>  [41] MatrixGenerics_1.8.1        fastmap_1.1.0              
>  [43] limma_3.52.4                rlang_1.0.6                
>  [45] RSQLite_2.2.18              generics_0.1.3             
>  [47] jsonlite_1.8.2              dplyr_1.0.10               
>  [49] RCurl_1.98-1.9              magrittr_2.0.3             
>  [51] GenomeInfoDbData_1.2.8      Matrix_1.5-1               
>  [53] Rcpp_1.0.9                  IRkernel_1.3               
>  [55] munsell_0.5.0               S4Vectors_0.34.0           
>  [57] fansi_1.0.3                 lifecycle_1.0.3            
>  [59] stringi_1.7.8               MASS_7.3-58.1              
>  [61] SummarizedExperiment_1.26.1 zlibbioc_1.42.0            
>  [63] plyr_1.8.7                  grid_4.2.1                 
>  [65] blob_1.2.3                  parallel_4.2.1             
>  [67] crayon_1.5.2                lattice_0.20-45            
>  [69] Biostrings_2.64.1           IRdisplay_1.1              
>  [71] splines_4.2.1               KEGGREST_1.36.3            
>  [73] pillar_1.8.1                GenomicRanges_1.48.0       
>  [75] uuid_1.1-0                  corpcor_1.6.10             
>  [77] reshape2_1.4.4              codetools_0.2-18           
>  [79] stats4_4.2.1                glue_1.6.2                 
>  [81] evaluate_0.17               EpiDISH_2.12.0             
>  [83] BiocManager_1.30.18         png_0.1-7                  
>  [85] vctrs_0.4.2                 foreach_1.5.2              
>  [87] locfdr_1.1-8                MatrixModels_0.5-1         
>  [89] gtable_0.3.1                purrr_0.3.5                
>  [91] tidyr_1.2.1                 reshape_0.8.9              
>  [93] assertthat_0.2.1            cachem_1.0.6               
>  [95] ggplot2_3.3.6               TOAST_1.10.1               
>  [97] xtable_1.8-4                e1071_1.7-11               
>  [99] coda_0.19-4                 class_7.3-20               
> [101] survival_3.4-0              tibble_3.1.8               
> [103] iterators_1.0.14            registry_0.5-1             
> [105] AnnotationDbi_1.58.0        memoise_2.0.1              
> [107] IRanges_2.30.1

**Comment by kphelan13** [comment]
Also am having the exact same error

**Comment by hasanalanya** [comment]
Same here, expression set not recognized. Jiaxin, we would appreciate it if you could fix the problem asap. @Jiaxin-Fan

**Comment by alexdray86** [comment]
Same problem here, unable to use MUSIC2, cannot find where this bulk.mtx is....

**Comment by hasanalanya** [comment]
> Same problem here, unable to use MUSIC2, cannot find where this bulk.mtx is....

Yes, even here is an error for the MuSiC2. Unable to find the new version of the MuSiC.

**Comment by Amit-H** [comment]
I am also having this error - any update on this would be greatly appreciated!

**Comment by spatts14** [comment]
Any updates? I am getting the same error.

**Comment by hasanalanya** [comment]
> Any updates? I am getting the same error.

I had the same problem over and over again and dont think that they maintain it
